docs/_documentations/metrics-dashboard.md

```diff
@@ -43,39 +43,32 @@ View these metrics to learn if your application is performing as intended. An ap
 
-## Clicking the **Run load** button
-
-Click the **Run load** button to run load your microservice. This button pings endpoints by using the Codewind load runner service.
-
 ## Viewing metrics information in the **Summary** tab
 
 View a summary of the information from the performance metrics graphs that you saw in the **Dashboard** tab.
 * **HTTP Requests:** This table displays how many times an endpoint is hit. If you repeatedly run a test while making code changes, you can see how the code changes are impacting application performance.
 * **Environment:** This table shows information about the environment you're running, such as the OS architecture and the number of processors you have. Having this information can make troubleshooting easier if you need to contact support and provide it.
-* **Resource usage:** This table shows the **Average Process CPU**, which shows what Codewind is doing, and the **Average System CPU**, which shows what the environment as a whole is doing. This information can help determine whether application issues are caused by something else in the environment that is external to the application.
+* **Resource Usage:** This table shows the **Average Process CPU**, which shows what Codewind is doing, and the **Average System CPU**, which shows what the environment as a whole is doing. This information can help determine whether application issues are caused by something else in the environment that is external to the application.
+
+![image of viewing metrics](dist/images/performanceguide/viewing_metrics.png)
 
 ## Viewing the graph in the **Profiling** tab
 
 The **Profiling** tab is available only in Node.js projects. This tab pulls information from the CPU metric in the **Dashboard** tab. CPU cycles are caused by an execution of code. Receiving an endpoint doesn't cause many CPU cycles, but receiving an incoming payload can. Profiling shows you what occurs when a spike appears in the CPU metric. Each spike on the flame graph shows a call stack in the path. The width of a spike indicates how much time the CPU spends on a function. Wide spikes indicate that your CPU spends much time on a particular function and where you might want change the code to optimize it.
 
 ## Profiling Data Support
 
-In Theia or VS Code, you can use the Profiling Language Server to provide code highlighting. Code highlighting displays the relative time spent in JavaScript functions based on profiling data gathered through [load testing](performance). Profiling support is only available for Node.js projects that are created through Codewind and then profiled.
+In Theia or VS Code, you can use the Profiling Language Server to provide code highlighting. Code highlighting displays the relative time spent in JavaScript functions based on profiling data gathered through [load testing](performancetesting). Profiling support is only available for Node.js projects that are created through Codewind and then profiled.
```

Comment:
I'm not sure this is right either. Code highlighting is implemented as a language server but knowing that is not much use to the user. They need to install the right VSCode extension to provide that for NodeJS or Java (Java is supported):

For Node apps they need to install: https://marketplace.visualstudio.com/items?itemName=IBM.codewind-node-profiler
For Java apps they need to install: https://marketplace.visualstudio.com/items?itemName=IBM.codewind-java-profiler

Java is supported, the confusing here may be that the metrics dashboard for Java doesn't show profiling data however the editor plugin can.

Reply:
Reworked this section.